src/components/pages/import.js

```diff
@@ -1,152 +1,149 @@
+            columns={attemtpsColumns}
+            pagination={false}
+            />
+        </Panel>
+      </Collapse>
+    );
   };
 
   const columns = [
-    { title: t('importType'), key: 'importType', width: '150px', sorter: by(x => x.importType), render: renderImportType },
-    { title: t('title'), key: 'title', sorter: by(x => x.title), render: renderTitle },
-    { title: t('language'), key: 'language', width: '150px', sorter: by(x => x.language), render: renderLanguage },
-    { title: t('updateDate'), key: 'updateDate', width: '200px', sorter: by(x => x.updatedOn), render: renderUpdateDate }
+    { title: t('importType'), dataIndex: 'taskParams', width: '50px', render: renderImportType },
+    { title: t('documentTitle'), key: '_id', dataIndex: 'taskParams', width: '150px', render: renderTitle },
+    { title: t('taskStatus'), dataIndex: 'processed', width: '150px', render: renderStatus },
+    { title: t('attemptsHeader'), dataIndex: 'attempts', width: '150px', render: attemptsRenderer }
   ];
 
-  const tableRowSelection = {
-    onChange: selectedRowKeys => {
-      setSelectedDocumentKeys(selectedRowKeys);
-    }
-  };
-
-  const importSourceOptions = sourceMenuItems.map(item => ({ label: item.name, value: item.name }));
-
   return (
     <Page>
-      <div className="ImportPage">
-        <h1>{t('pageNames:import')}</h1>
-
-        <Select
-          placeholder={t('source')}
-          className="ImportPage-source"
-          onChange={handleSourceMenuChange}
-          defaultValue={null}
-          disabled={isFetchingImportableDocuments}
-          options={importSourceOptions}
-          />
-        <br /> <br />
-
+      <div className="CreateImportPage">
+        <h1>{t('importHeading')}</h1>
+        <Row>
+          <Space>
+            <span>{t('batchStatus')}:</span>
+            <span>{renderBatchStatus(batch)}</span>
+          </Space>
+        </Row>
+        <Row>
+          <Space>
+            <span>{t('createdOn')}:</span>
+            <span>{renderDate(batch.createdOn)}</span>
+          </Space>
+        </Row>
+        <Row>
+          <Space>
+            <span>{t('completedOn')}:</span>
+            <span>{renderDate(batch.completedOn)}</span>
+          </Space>
+        </Row>
+        <Row>
+          <Space>
+            <span>{t('createdBy')}:</span>
+            <span>{batch.createdBy.username}</span>
+          </Space>
+        </Row>
+        <Collapse>
+          <Panel header={t('batchErrors')} extra={getErrorsExtra(batch)}>
+            <List
+              dataSource={batch.errors}
+              renderItem={error => (
+                <List.Item>
+                  <span>{error}</span>
+                </List.Item>)}
+              />
+          </Panel>
+        </Collapse>
         <Table
-          bordered
-          size="small"
-          rowSelection={{
-            type: 'checkbox',
-            ...tableRowSelection
-          }}
+          style={{ marginTop: '20px' }}
```

Comment:
I wanted to use an expandable table here, but the expand button wouldn't appear... So I went the collapse panel variant, not bad but not great either.

Reply:
Since you style everything else with classes here, you could maybe throw this style in a class too.

Reply:
Sure